features/support-form/components/SupportForm.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import React from "react";
+import { Button } from "components/Button";
+import { Dialog } from "components/Dialog";
+import { Input } from "components/Input";
+import { TextArea } from "components/TextArea";
+
+export function SupportForm() {
+  return (
+    <Dialog title="Potrzebujesz pomocy?" description="Odpiszemy w ciągu 24 godzin" button={<button className="absolute bottom-4 right-4">Potrzebuję pomocy</button>}>
+      <form className="grid gap-5">
+        <Input type="text" name="imie" placeholder="Zuzanna" label="Imię" />
+        <Input
+          type="text"
+          name="nazwisko"
+          placeholder="Jurczak"
```

Comment:
Pani Zuzanna jest już po ślubie - ma teraz na nazwisko Grabowska.
```suggestion
          placeholder="Grabowska"
```